package.json

```diff
@@ -52,7 +52,7 @@
     "mini-css-extract-plugin": "^0.9.0",
     "mobius1-selectr": "^2.4.13",
     "mocha": "^7.1.1",
-    "node-sass": "^4.13.1",
+    "node-sass": "^4.14.1",
```

Comment:
@NickB64 I thought you guys made this update already on the previous group PR for the UK. Maybe update the repro you are working from so it's most up-to-date?

Reply:
I think when we submitted our PR from Steph's forked repo, we had an updated version of SASS but the PR we merged into the master branch was from my branch which I don't think I updated the SASS version of.